src/components/link/link.test.tsx

```diff
@@ -0,0 +1,570 @@
+// We've chosen to ignore this for the whole file as this warning fires 15 times.
+// It warns that `... anchors are primarily expected to navigate. Use the button element instead.`
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
**comment(non-blocking)**: we have a ticket on our backlog to remove polymorphism from `Link` as an accessibility improvement. What do you think about referencing this here?

```suggestion
/* TODO: FE-6579 To re-enable once button-related props are removed from Link */
/* eslint-disable jsx-a11y/anchor-is-valid */
```

Reply:
Yeah, I had no idea we already had a ticket would resolve this issue. I can change the comment to that which you have provided.